script.js

```diff
@@ -0,0 +1,92 @@
+function isValid(btn) {
+  return btn.innerText.length == 0;
+}
+
+function checkForWinner (squares, players, currentTurn) {
+  if (squares [0].innerText == players[currentTurn] &&
+      squares [1].innerText == players[currentTurn] &&
+      squares [2].innerText == players[currentTurn]) {
+      return true;
+  }else if (squares [3].innerText == players[currentTurn] &&
+      squares [4].innerText == players[currentTurn] &&
+      squares [5].innerText == players[currentTurn]){
+      return true;
+  }else if (squares [6].innerText == players[currentTurn] &&
+      squares [7].innerText == players[currentTurn] &&
+      squares [8].innerText == players[currentTurn]){
+      return true;
+  }else if (squares [0].innerText == players[currentTurn] &&
+      squares [3].innerText == players[currentTurn] &&
+      squares [6].innerText == players[currentTurn]) {
+      return true;
+  }else if (squares [1].innerText == players[currentTurn] &&
+      squares [4].innerText == players[currentTurn] &&
+      squares [7].innerText == players[currentTurn]) {
+      return true;
+  }else if (squares [2].innerText == players[currentTurn] &&
+      squares [5].innerText == players[currentTurn] &&
+      squares [8].innerText == players[currentTurn]) {
+      return true;
+  }else if (squares [0].innerText == players[currentTurn] &&
+      squares [4].innerText == players[currentTurn] &&
+      squares [8].innerText == players[currentTurn]) {
+      return true;
+  }else if (squares [2].innerText == players[currentTurn] &&
+      squares [4].innerText == players[currentTurn] &&
+      squares [6].innerText == players[currentTurn]) {
+      return true;
+  }
+}
+
+function setMark(btn, mark) {
+  btn.innerText = mark;
+}
+
+function isBoardFull(squares) {
+  for (var i = 0, i < squares.length; i++)
```

Comment:
For formatting here is off. There should be a semicolon `;` after the `var i = 0`, not a comma. Also, you're missing curly braces around your `for` loop.